src/draw/DefaultDrawer.js

```diff
@@ -46,6 +32,49 @@ class DefaultDrawer {
      * @type {Object}
      */
     this.resources = resources;
+
+    /**
+     * Render svg viewbox width
+     * @type {number}
+     */
+    this.width = options.width || 1280;
+
+    /**
+     * Render svg viewbox height
+     * @type {number}
+     */
+    this.height = options.height || 1280;
+
+    /**
+     * Dictionary of subjects to send events to.
+     * @type {Object}
+     */
+    this.events = events
+
+    /**
+     * Minimum height of a component
+     * @type {number}
+     */
+    this.baseHeight = options.baseHeight || 50;
```

Comment:
why not `minHeight` ? where does the object `options` come from, created in core or retrieve from d3 ?

Reply:
Renamed into minHeight.
Options object can be passed when instanciating the Drawer class to customize the layout and appearance of the model view.